go/test/endtoend/messaging/message_test.go

```diff
@@ -236,6 +233,75 @@ func TestThreeColMessage(t *testing.T) {
 	assert.Equal(t, uint64(1), qr.RowsAffected)
 }
 
+var createAllowedStreamingColsMessage = `create table vitess_message4(
+	id bigint,
+	priority bigint default 0,
+	time_next bigint default 0,
+	epoch bigint,
+	time_acked bigint,
+	msg1 varchar(128),
+	msg2 bigint,
+	primary key(id),
+	index next_idx(priority, time_next desc),
+	index ack_idx(time_acked))
+comment 'vitess_message,vt_stream_cols=id;msg1,vt_ack_wait=1,vt_purge_after=3,vt_batch_size=2,vt_cache_size=10,vt_poller_interval=1'`
```

Comment:
Don't feel strongly, but I think `vt_message_cols` would be better since this applies to the messages at a higher level, and at the lower level applies to both the poller and the results stream.

And beyond that ~ `createLimitedStreamingColsMessage` or ~ `TestSpecificStreamingColsMessage` would be slightly better IMO. Also minor things that I don't feel strongly about.

Reply:
done